azurerm/import_arm_operational_insight_workspace_test.go

```diff
@@ -0,0 +1,56 @@
+package azurerm
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAzureRMOperationalInsightWorkspace_importrequiredOnly(t *testing.T) {
```

Comment:
_minor_ we'd tend to name this `_importRequiredOnly` - could we update it to match?

Reply:
Done!